src/utils/env_helper.ts

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
nitpick - can you remove the whitespace up here

Reply:
yep, will do